index.html

```diff
@@ -17,6 +17,7 @@ <h1>Headphones recommended</h1>
   <canvas class="background"></canvas>
   <div class="greeting main-greeting">Have a peaceful 2021</div>
   <div class="greeting secondary-greeting">from Nishant, Varsha & Mahi</div>
+  <div class="mute-audio">Mute</div>
```

Comment:
Using divs as buttons is bad. Use native buttons.

Reply:
Incorporated